### PR TITLE
#4 lista servidores

### LINKED ISSUE #4: Agregar lista servidores preferidos
En el menu conectar tenemos que crear una lista donde se guarden todos los servidores que vamos entrando/agregando, permitiendo al usuario hacer click sobre un servidor en la lista y cambiar los parametros de conexion para poder conectarse al servidor elegido.

Se deberan guardar los siguientes parametros:

    Nombre servidor.
    Puerto.
    Ip

Guardar/Leer lista en el archivo INIT/sinfo.dat

Verificar el uso del parametro P2 y si esta en deshuso eliminar. (creo que tiene algo que ver con estadisticas del servidor)

Leer SINFO.dat desde github

https://raw.githubusercontent.com/ao-libre/ao-cliente/master/INIT/sinfo.dat

y la gente podria agregar su servidor a la lista mandando un PULL REQUEST

### LINKED ISSUE #48: Invalid character in "Welcome" message
CODIGO/General.bas
Line 953

Call AddtoRichTextBox(frmCargando.Status, "                    Â¡Bienvenido a Argentum Online!"

### PR DESCRIPTION
**Pull request by cucsijuan**
Closes #4 
Closes #48 
Added a list box on connect form to store server IPs downloaded from internet
Added a Sub to download such info, it uses Microsoft ActiveX Data Objects 2.5 Library
PassRecPort property was deleted since there was no use for it
Ventanaconectar.jpg now have a box to hold the server list

### CODE REVIEW

**Comment by cucsijuan** [comment]
Habiendo investigado bien el tema descubrí que también utiliza internet explorer xmlhttp asi que no encontrando ningún beneficio y para tener consistencia voy a pasarlo a INet1

**Comment by cucsijuan** [comment]
Estoy usando INet, aunque tiene un manejo de errores feo hice que funcione como la libreria anterior, si tiene algun problema al levantar la URL, no baja el archivo y continua con la carga del que tenemos en el disco.

Im using INet now, it doesn't have a very good error handling system though, but i manage to make it behave like the XMLHTTP. When there's some issue with the URL, the program resumes using the file we have in the HD.

This is the new interface: 
image

Suggestions are welcome :)

**Comment by juandpr** [comment]
I love it. Approved

File: CODIGO/General.bas [hunk 1]
```diff
@@ -923,34 +931,34 @@ Private Sub LoadInitialConfig()
...
     'Inicializamos el inventario grafico
-    Call Inventario.Initialize(DirectD3D8, frmMain.picInv, MAX_INVENTORY_SLOTS)
+    Call Inventario.Initialize(DirectD3D8, frmMain.PicInv, MAX_INVENTORY_SLOTS)
     'Call Audio.MusicMP3Play(App.path & "\MP3\" & MP3_Inicio & ".mp3")
-    Call AddtoRichTextBox(frmCargando.Status, "Hecho", 255, 0, 0, True, False, False)
+    Call AddtoRichTextBox(frmCargando.status, "Hecho", 255, 0, 0, True, False, False)
     
     
-    Call AddtoRichTextBox(frmCargando.Status, "                    ¡Bienvenido a Argentum Online!", 255, 255, 255, True, False, True)
+    Call AddtoRichTextBox(frmCargando.status, "                    ¡Bienvenido a Argentum Online!", 255, 255, 255, True, False, True)
```
[the remark this answers is not in the document]
Comment (RecoX) [reply CODIGO/General.bas]: Yes, it was added when i changed the encoding, i need to fix it.

**Comment by cucsijuan** [comment]
Fixes #48

**Pull request merged**